wrapper/src/main/java/software/amazon/jdbc/plugin/readwritesplitting/ReadWriteSplittingPlugin.java

```diff
@@ -370,7 +373,7 @@ private void logAndThrowException(final String logMessage, final SqlState sqlSta
   private void logAndThrowException(
       final String logMessage, final SqlState sqlState, final Throwable cause)
       throws SQLException {
-    LOGGER.severe(logMessage);
+    LOGGER.fine(logMessage);
```

Comment:
Is the intent to eliminate potential double logging, as this exception is likely to be caught and logged upstream?

Reply:
I think we might scare users by logging SEVERE message. Severe isn't a proper level of error. It's not a critical error, the plugin can handle it properly and continue with normal workflow. So, I think that FINE would be appropriate level here.